packages/ckeditor5-list/tests/listformatting/listitemfontfamilyintegration.js

```diff
@@ -72,15 +73,33 @@ describe( 'ListItemFontFamilyIntegration', () => {
 	} );
 
 	describe( 'schema', () => {
-		it( 'should allow listItemFontFamily attribute in $listItem', () => {
-			expect( model.schema.checkAttribute( [ '$root', '$listItem' ], 'listItemFontFamily' ) ).to.be.true;
-		} );
-
```

Comment:
This makes sure that all elements inheriting from it will support that attribute so I'd keep it.